sway-lsp/src/server.rs

```diff
@@ -13,12 +14,20 @@ use tower_lsp::{jsonrpc, Client, LanguageServer};
 pub struct Backend {
     pub client: Client,
     session: Arc<Session>,
+    debug: debug::DebugFlags,
 }
 
 impl Backend {
     pub fn new(client: Client) -> Self {
         let session = Arc::new(Session::new());
-        Backend { client, session }
+        let debug = debug::DebugFlags {
```

Comment:
Could potentially derive `Default` for `DebugFlags` and use `debug::DebugFlags::default();` here

Reply:
I did think of doing that. This way though it makes it really quick while developing to just set the variable to `true` and then back to `false` when you're finished.